include/ListView/ListViewSubPanel.php

```diff
@@ -504,7 +504,13 @@ public function process_dynamic_listview_rows($data, $parent_data, $smartyTempla
                                         unset($_content);
                                     }
                                     else {
-                                        $button_contents[$aVal][] = '';
+                                        $dontProcessThisFields = array("edit_button", "close_button","remove_button");
```

Comment:
The word "fields" in SuiteCRM means something you can store in database. The word "actions" in SuiteCRM means something you can do. Since the array stores actions, can you please this name to something like "$doNotProcessTheseActions"?